src/components/utils/componentImage.js

```diff
@@ -0,0 +1,18 @@
+const getProductVariablesByProdname = require('../../utils/getProductVariablesByProdname');
```

Comment:
We need to use CJS modules here because this file is used in a plugin that is executed on build time. Build time uses CJS modules - it's Docusaurus default configuration so I'm not sure we can switch to ESM modules quite easily.

Reply:
Noted, thanks